docs/get-started/Validator-Node.md

```diff
@@ -26,21 +26,21 @@ It is the only way to recover your account if you ever forget your password.
 blast change tumble toddler rival ordinary chicken dirt physical club few language noise oak moment consider enemy claim elephant cruel people adult peanut garden
 ```
 
-You could see the address and public key of this account. Please node that account address in IRISnet will start with `iaa` and public key of account will start with `iap`.
+You could see the address and public key of this account. Please notice that account address in IRISnet will start with `iaa1` and public key of account will start with `iap1`.
 
 The seed phrase of this account will also be displayed. You could use these 24 phrases to recover this account in another server. The recover command is:
 ```
-iriscli keys add <NAME_OF_KEY> --recover
+iriscli keys add <key_name> --recover
 ```
 
 
-### Claim tokens
+### Claim tokens (Only for fuxi Testnet)
 
-You can always get some `IRIS`  by using the [Faucet](https://testnet.irisplorer.io/#/faucet). The faucet will send you 10IRIS every request, Please don't abuse it.
+You can always get some `IRIS`  by using the [Faucet](https://testnet.irisplorer.io/#/faucet). The faucet will send you 10IRIS for every request, please don't abuse it.
```

Comment:
You can always get some test tokens by using ...

Reply:
fixed